fem/hybridization_ext.cpp

```diff
@@ -299,16 +306,28 @@ void HybridizationExtension::ConstructH()
    Vector AhatInvCt_mat;
 
    {
+      // The dispatch below is based on the following sizes, sorted
+      // appropriately.
+      //
+      // RT(k) in 2D (quads): (interior,boundary) dofs:
+      // - arbitrary k: 2*(k+1)*(k+2)-4*(k+1), 4*(k+1)
+      // - k=0: (0,4)
+      // - k=1: (4,8)
+      // - k=2: (12,12)
+      // - k=3: (24,16)
+      // RT(k) in 3D (hexes): (interior,boundary) dofs:
+      // - arbitrary k: 3*(k+1)^2*(k+2)-6*(k+1)^2, 6*(k+1)^2
+      // - k=0: (0,6)
+      // - k=1: (12,12)
+      // - k=2: (54,54)
       const int NI = idofs.Size();
       const int NB = bdofs.Size();
-      // 2D
       if (NI == 0 && NB <= 4) { FactorElementMatrices<0,4>(AhatInvCt_mat); }
+      else if (NI == 0 && NB <= 6) { FactorElementMatrices<0,6>(AhatInvCt_mat); }
       else if (NI <= 4 && NB <= 8) { FactorElementMatrices<4,8>(AhatInvCt_mat); }
       else if (NI <= 12 && NB <= 12) { FactorElementMatrices<12,12>(AhatInvCt_mat); }
-      else if (NI <= 24 && NB <= 16) { FactorElementMatrices<12,16>(AhatInvCt_mat); }
-      // 3D
-      else if (NI <= 0 && NB <= 6) { FactorElementMatrices<0,6>(AhatInvCt_mat); }
       else if (NI <= 12 && NB <= 24) { FactorElementMatrices<12,24>(AhatInvCt_mat); }
```

Comment:
@pazner, what was the motivation to include the case `(12,24)`? It does not correspond to RT space on quads/hexes, according to the numbers in the code comment above.

Reply:
You are right! I somehow ended up copying the wrong pair of numbers when I computed these in python.

I'll update the comment.